File: system-test/dotnet/SystemTests/Latest/SmokeTests/External/TaxSmokeTest.cs
using SystemTests.Latest.Base;
using Xunit;

namespace SystemTests.Latest.SmokeTests.External;

public class TaxSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToTax()
    {
        await Scenario().Assume().Tax().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/latest/smoke/external/TaxSmokeTest.java
package com.mycompany.myshop.systemtest.latest.smoke.external;

import com.mycompany.myshop.systemtest.latest.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class TaxSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToTax() {
        scenario.assume().tax().shouldBeRunning();
    }
}
